cmd/lotus-storage-miner/retrieval-deals.go

```diff
@@ -234,8 +234,9 @@ var retrievalSetAskCmd = &cli.Command{
 }
 
 var retrievalGetAskCmd = &cli.Command{
-	Name:  "get-ask",
-	Usage: "Get the provider's current retrieval ask",
+	Name: "get-ask",
+	Usage: "Get the current retrieval ask configured by the provider in the ask-store. No-op if user has configured an external script to " +
+		"dynamically price retrieval deals.",
```

Comment:
Why a no-op in this case? The external script should also receive the configured ask, that way the system is symmetric in all its possible pricing function strategies.